script/Arcueid/Day 5/05_00_ARC05_2_2.txt

```diff
@@ -167,22 +167,22 @@ But still――― Even a person like me can say what I don't want to do.
 [sha:5d54ff30df25ddd1d30cd4870a6c815d2036cbd8]{
 -- Page 213, Offset 9123.
 -- 「……アルクェイド。おまえの話は、俺には分からない。
-"...Arcueid. I don't understand what you're talking about.
+"...Arcueid. I don't get what you're talking about.
 }
 [sha:17b851d0690052eb46aaf68ed7d47c5d7e404ebd]{
 -- Page 213, Offset 9124.
 -- どんなに相手が違う生き物だとしても、どんなに俺が無力だとしても、責任は同じなんだ。俺はやっぱり、多くの人を見殺しにしたし、多くの人を見殺しにするのはイヤなんだ。
-No matter how much of a monster he is, no matter how powerless I am, my responsibility is still the same. As I thought, I let many people die, and I don't want to see anyone else meet that fate.
+No matter how much of a monster he is, no matter how powerless I am, my responsibility is still the same. Just as I thought, I let many people die, and I don't want to see anyone else meet that fate.
```

Comment:
Rather than 'just as I thought' I think something like 'Regardless of how you put it,' fits better